client/apps/settings/index.coffee

```diff
@@ -0,0 +1,14 @@
+#
+# CRUD for curations
+#
+
+express = require 'express'
+routes = require './routes'
+
+app = module.exports = express()
+app.set 'views', __dirname
+app.set 'view engine', 'jade'
+
+app.get '/settings', routes.index
+app.get '/settings/:id/edit', routes.edit
+app.post '/settings/:id', routes.save
```

Comment:
Nah, given the UI I think this makes sense. Typically for 1:1 "CRUD" views it's good to keep that "RESTful" naming convention, but not a hard fast rule.